grpcdogfood/server/chart/templates/deployment.yaml

```diff
@@ -0,0 +1,81 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Values.fullname }}
+  {{- with .Values.namespace }}
+  namespace:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
```

Comment:
Todo: pass in the port programmatically (I think I need a `container` to do this, not sure when this `deployment.yaml` diverged from the client one.